File: 23 mar 22/calculation.py

class calculation :
    
    def __init__(self,ino1,ino2):
        self.i=ino1
        self.j=ino2
        
    def sum(self):
        ans=self.i+self.j
        print("Sumetion is : " ,ans)
        
    def sub(self):
        ans=self.i-self.j
        print("subtraction is :" ,ans)
        
    def div(self):
        ans=self.i/self.j
        print("Division is :" , ans)
        
    def mult(self):
        ans=self.i*self.j
        print("Multiplication is :",ans)

def main():
    a=int(input("Enter first Number :"))
    b=int(input("Enter first Number :"))
    
    obj=calculation(a,b)
    

    obj.sum()
    obj.sub()
    obj.div()
    obj.mult()


if __name__ == "__main__":
    main()

File: 23 mar 22/calculation.js


class calculation
{
	constructor (no1,no2)
	{
		this.no1=no1;
		this.no2=no2;
	}
	
	sum()
	{
		var ans=0;
		ans=this.no1+this.no2
		console.log("Summetion is :" +ans);
	}
	
	sub()
	{
		var ans=0;
		ans=this.no1-this.no2
		console.log("Subtraction is :" +ans);
	}
	
	div()
	{
		var ans=0;
		ans=this.no1/this.no2
		console.log("division is :" +ans);
	}
	
	mult()
	{
		var ans=0;
		ans=this.no1*this.no2
		console.log("multiplication is :" +ans);
	}
}

const obj1 = new calculation(10,5);
const obj2 = new calculation(40,5);
obj1.sum();
obj1.sub();
obj1.div();
obj1.mult();

obj2.sum();
obj2.sub();
obj2.div();
obj2.mult();

